rdkit/Chem/Fraggle/FraggleSim.py

```diff
@@ -165,13 +162,11 @@ def select_fragments(fragments, ftype, hac):
     # appropriate fragmentation must have 2 components and needs to be greater than 60% of
     # parent mol
     if len(result) == 2 and result_hcount > 0.6 * hac:
-      result = '.'.join(result)
-    else:
-      result = None
-    return result
+      return '.'.join(result)
+    return None
 
-  else:
-    raise NotImplementedError('Invalid fragmentation type {0}'.format(ftype))
+  # This should never happen
```

Comment:
The comment is redundant and potentially misleading. NotImplementedError says that an implementation is missing and would need to be added

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.